components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/NimbusInterface.kt

```diff
@@ -224,4 +213,53 @@ interface NimbusInterface : FeaturesInterface, GleanPlumbInterface {
     }
 }
 
+interface NimbusEventStore {
+    /**
+     * Records an event to the Nimbus event store.
+     *
+     * The method obtains the event counters for the `eventId` that is passed in, advances them if
+     * needed, then increments the counts by 1. If an event counter does not exist for the `eventId`,
+     * one will be created.
+     *
+     * @param eventId string representing the id of the event which should be recorded.
+     */
+    fun recordEvent(count: Long = 1, eventId: String) = Unit
```

Comment:
This will be a breaking change for Fenix. They'll need to update the `recordEvent` calls to include the parameter name for eventId.

https://github.com/mozilla-mobile/firefox-android/blob/610e1ad7fc901ef2307b8b4851b293c87d9791c3/fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt#L302

```kotlin
components.analytics.experiments.recordEvent("app_opened")
// becomes
components.analytics.experiments.recordEvent(eventId = "app_opened")
```

Reply:
Realized just now but the same applies to iOS with the swift updates

Reply:
Ok, I added more API that conforms to the existing API.